L1TriggerConfig/CSCTFConfigProducers/src/L1MuCSCPtLutConfigOnlineProd.cc

```diff
@@ -24,7 +24,7 @@ L1MuCSCPtLutConfigOnlineProd::newObject( const std::string& objectKey )
     {
       edm::LogError( "L1-O2O" ) << "Problem with L1MuCSCPtLutParameters key" ;
       // return empty object
-      return std::shared_ptr< L1MuCSCPtLut >() ;
+      return std::make_shared< L1MuCSCPtLut >() ;
```

Comment:
These are different. The original created an empty `shared_ptr`. Your code makes a `shared_ptr` containing a defaultly constructed object.

Please set the code back to what it was.